lib/location/location.c

```diff
@@ -128,17 +171,20 @@ static void location_config_method_defaults_set(
 
 	method->method = method_type;
 	if (method_type == LOCATION_METHOD_GNSS) {
-		method->gnss.timeout = 120 * MSEC_PER_SEC;
-		method->gnss.accuracy = LOCATION_ACCURACY_NORMAL;
-		method->gnss.num_consecutive_fixes = 3;
-		method->gnss.visibility_detection = false;
-		method->gnss.priority_mode = false;
+		method->gnss.timeout = CONFIG_LOCATION_REQUEST_DEFAULT_GNSS_TIMEOUT;
+		method->gnss.accuracy = CONFIG_LOCATION_REQUEST_DEFAULT_GNSS_ACCURACY_NORMAL;
```

Comment:
This probably doesn't work as intended, the configuration needs to be converted to proper enum values.

Reply:
True, will fix